src/lib/viewers/BaseViewer.js

```diff
@@ -44,17 +47,16 @@ const VIEWER_STATUSES = {
 const ANNOTATIONS_JS = 'annotations.js';
 const ANNOTATIONS_CSS = 'annotations.css';
 
+const language = __LANGUAGE__; // eslint-disable-line
 const ANNOTATION_TYPE_DRAW = 'draw';
 const ANNOTATION_TYPE_POINT = 'point';
 const LOAD_TIMEOUT_MS = 180000; // 3m
 const RESIZE_WAIT_TIME_IN_MILLIS = 300;
 const ANNOTATION_BUTTONS = {
     point: {
-        title: __('annotation_point_toggle'),
```

Comment:
I think these are still needed, as they are part of Preview rather than Annotations.